docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -59,3 +59,38 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.2.
 {{% /admonition %}}
+
+### OAuth role mapping enforcement
+
+<!-- AuthNZ -->
+
+This change impacts `GitHub` OAuth, `Gitlab` OAuth, `Okta` OAuth and `Generic` OAuth.
+
+Currently if no organization role mapping is found for a user when connecting via OAuth, Grafana doesn’t update the user’s organization role.
+
+With Grafana 10.1, on every login, if the role_attribute_path property does not return a role, then the user is assigned the role specified by the auto_assign_org_role option or the default role for the organization, by default, Viewer.
+
+To avoid overriding manually set roles, enable the skip_org_role_sync option in the Grafana configuration for your OAuth provider before the user logs in for the first time.
+
+### GitLab OIDC support
+
+<!-- AuthNZ -->
+
+Grafana 10.1 now supports GitLab OIDC via the `GitLab` OAuth provider in addition to the existing `GitLab` OAuth2 provider. This allows you to use GitLab OIDC to authenticate users in Grafana.
+
+This allows Grafana to reduce the access scope to only the required scopes for authentication and authorization instead
+of full read API access.
+
+To learn how to migrate your GitLab OAuth2 setup to OIDC, refer to our [GitLab authentication documentation]({{< relref "../setup-grafana/configure-security/configure-authentication/gitlab/" >}}).
+
+### Google OIDC and teamsync support
+
+<!-- AuthNZ -->
+
+Grafana 10.1 now supports Google OIDC via the `Google` OAuth provider in addition to the existing `Google` OAuth2 provider. This allows you to use Google OIDC to authenticate users in Grafana.
+
+This allows Grafana to reduce the access scope to only the required scopes for authentication and authorization.
+
+This release also adds support for Google OIDC in Team Sync. You can now easily add users to teams by utilizing their Google groups.
+
+To learn how to migrate your Google OAuth2 setup to OIDC and how to setup teamsync, refer to our [Google authentication documentation]({{< relref "../setup-grafana/configure-security/configure-authentication/google/" >}}).
```

Comment:
Nit: I'm not actually sure what our preferred way of referring to team sync is, but it would be good to at least be consistent within the section. I'd go with `Team Sync` (and also update the section header).

```suggestion
To learn how to migrate your Google OAuth2 setup to OIDC and how to setup Team Sync, refer to our [Google authentication documentation]({{< relref "../setup-grafana/configure-security/configure-authentication/google/" >}}).
```